tiledb/sm/tile/tile.cc

```diff
@@ -249,7 +249,7 @@ Status Tile::zip_coordinates() {
   return Status::Ok();
 }
 
-uint64_t Tile::load_chunk_data(ChunkData& unfiltered_tile) {
+uint64_t Tile::load_chunk_data(ChunkData& unfiltered_tile, bool is_offsets) {
```

Comment:
My only substantive comment on first review is to ask whether this might be better as two functions that hide the boolean argument. Something like `load_chunk_data` and `load_offsets_chunk_data`? The few places we call this I couldn't keep the polarity of the boolean straight so I had to reference this a few extra times. Certainly not a huge deal either way, just one of those things that might help long term maintenance work.